### DISCUSSION TITLE
Open Side Menu

### DISCUSSION THREAD

**Discussion by choskins-godaddy** [bootscore/bootscore#336] [Q&A] [answered]
Could someone please explain this "Open Side Menu" button that opens a blank sidebar on xs screens?  This isn't one of the menu's I have configured.

**Comment by crftwrk** [comment, marked as answer]
That's not one of your menus, it's the sidebar with widgets. If you haven't any active widgets in the sidebar, choose a page/post template without sidebar in the backend to hide this button.

- https://bootscore.me/documentation/page-post-templates/
- https://bootscore.me/documentation/loop-templates/

| xs | ≥md |
| ------------- | ------------- |
| ![sidebar-xs](https://user-images.githubusercontent.com/51531217/209324371-ed0790eb-a1b1-402d-af95-cacc010c524e.png) | ![sidebar](https://user-images.githubusercontent.com/51531217/209324455-a05ce1ba-d5c9-4974-9921-8ce67015aa62.png)  |

I agree that name "Open Side Menu" instead of "Open Sidebar" is a bit confusing, but we decided to name it that way for this 
https://github.com/bootscore/bootscore/pull/171#issuecomment-1155429407 reason.

Does this clear it up?

**Reply by choskins-godaddy** [reply]
Hmmm, this is in the Product page and the Category pages.  I'm not sure where to override that in my child theme.  I'm assuming it's content-product.php, and content-product-cat.php but I don't see where the page template is set.

**Reply by crftwrk** [reply]
No, it is woocommerce.php and sidebar is dynamic. Means if you have widgets active in sidebar position, then button is visible. If you want to hide sidebar in WooCommerce pages, then you must hide widgets there. Check out this issue how to hide/show sidebar in different pages https://github.com/bootscore/bootscore/issues/150

**Reply by choskins-godaddy** [reply]
Brilliant.  Thank you.
